apps/sample-app/src/App.tsx

```diff
@@ -20,12 +28,17 @@ const App: React.FC = () => {
         app.notifyAppLoaded();
         app.notifySuccess();
         const context = await app.getContext();
-        const themeNow = getTheme(context?.app?.theme);
-        setCurrTheme(themeNow);
-        app.registerOnThemeChangeHandler(function(theme) {
-          setCurrTheme(getTheme(theme));
-        });
+        if (context?.app?.host?.name === 'Teams') {
```

Comment:
@MarcinJuraszek What do you think about a comment saying a disclaimer where it is ok to check for specific host name for details like theme but that we would not recommend checking the name to see if a capability is supported, and to simply use isSupported() calls in that case?

Reply:
How about a link to https://docs.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/using-teams-client-sdk?tabs=javascript%2Cmanifest-teams-toolkit#differentiate-your-app-experience in a comment?